src/BlockTypes/Cart.php

```diff
@@ -48,6 +48,9 @@ public function render( $attributes = array(), $content = '' ) {
 			$this->block_name . '-frontend',
 			$this->block_name . '-block-frontend'
 		);
+		\Automattic\WooCommerce\Blocks\Assets::register_block_script(
+			'vendors-frontend'
+		);
```

Comment:
Shouldn't this be enqueued before the block scripts?

Also, at some point (we could create an issue for now) we'll likely want to abstract this so that we have some way of automatically adding the vendors-frontend script as a dependency if needed (might be a way to do this for both `vendors` and `vendors-frontend` actually).

Reply:
> Shouldn't this be enqueued before the block scripts?

Sure! Actually I think it makes sense to define it as a dependency of the `cart-frontend.js` script. Done in 064a4532ab96624449ab9f90cdda97530253854e.